include/diaspora.php

```diff
@@ -932,18 +932,20 @@ private static function post_allow($importer, $contact, $is_comment = false) {
 			logger("defining user ".$contact["nick"]." as friend");
 		}
 
-		// We don't seem to like that person
+		// Is this contact wanted?
```

Comment:
I think that @rabuzarus want to say that the "we" style is okay and needn't to be neutralized - but this specific sentence seems to be misleading, so it could be changed.